### PR TITLE
Updated Bulgarian Translation

### PR DESCRIPTION
**Pull request by RacerBG**

### CODE REVIEW

File: src/Translations/Ingame.csv [hunk 1]
```diff
@@ -51,13 +51,13 @@ TRAIN_INFO_ENFORCED_BREAKING,"Enforced Breaking (check engine, red signal, speed
...
-TRAIN_INFO_ACCELERATION_1,"Acceleration (Apply: '%s', Release: l'%s', check doors)","Beschleunigung (Erhöhen: '%s', Verringern: '%s'; Überprüfen: Türen)",,力行 (‘%s’上矢印キー、‘%s’右矢印キー),"Gaz (przyspiesz: '%s', zwolnij: '%s', sprawdż drzwi)","Прискорення (додати: ‘%s’; відпустити: ‘%s’; перевірте двері)","Aceleración (Aplicar: '%s', Liberar: l'%s', comprobar las puertas)","Ускорение (прилагане: „%s“, освобождаване: „%s“, проверете вратите)","Accélération (Appliquer : '%s', Libérer : l'%s', vérifier les portes)"
+TRAIN_INFO_ACCELERATION_1,"Acceleration (Apply: '%s', Release: l'%s', check doors)","Beschleunigung (Erhöhen: '%s', Verringern: '%s'; Überprüfen: Türen)",,力行 (‘%s’上矢印キー、‘%s’右矢印キー),"Gaz (przyspiesz: '%s', zwolnij: '%s', sprawdż drzwi)",Прискорення (додати: ‘%s’; відпустити: ‘%s’; перевірте двері),"Aceleración (Aplicar: '%s', Liberar: l'%s', comprobar las puertas)","Ускорение (прилагане: „%s“, освобождаване: „%s“, проверете вратите)","Accélération (Appliquer : '%s', Libérer : l'%s', vérifier les portes)"
```
Comment (nalquas) [inline src/Translations/Ingame.csv]:
```suggestion
TRAIN_INFO_ACCELERATION_1,"Acceleration (Apply: '%s', Release: l'%s', check doors)","Beschleunigung (Erhöhen: '%s', Verringern: '%s'; Überprüfen: Türen)",,力行 (‘%s’上矢印キー、‘%s’右矢印キー),"Gaz (przyspiesz: '%s', zwolnij: '%s', sprawdż drzwi)","Прискорення (додати: ‘%s’; відпустити: ‘%s’; перевірте двері)","Aceleración (Aplicar: '%s', Liberar: l'%s', comprobar las puertas)","Ускорение (прилагане: „%s“, освобождаване: „%s“, проверете вратите)","Accélération (Appliquer : '%s', Libérer : l'%s', vérifier les portes)"
```

File: src/Translations/Ingame.csv [hunk 2]
```diff
@@ -51,13 +51,13 @@ TRAIN_INFO_ENFORCED_BREAKING,"Enforced Breaking (check engine, red signal, speed
...
 TRAIN_INFO_FRONT_LIGHT,Front Light ('%s'),Spitzenlicht ('%s'),Luci frontali (‘%s’),前方ライト (‘%s’),"Przednie światłła (""%s"")",Переднє світло (‘%s’),Luces frontales ('%s'),Предни светлини („%s“),Feu avant ('%s')
 TRAIN_INFO_CABIN_LIGHT,Cabin Light ('%s'),Kabinenlicht ('%s'),Luci cabina (‘%s’),室内電気 ('C%s'),"Światła Kabiny (""%s"")",Світло в кабіні ('%s'),Luces de cabina ('%s'),Осветление („%s“),Lumière de la cabine ('%s')
 TRAIN_INFO_AUTOPILOT,Autopilot ('%s'),Autopilot ('%s'),Guida automatica (‘%s’),自動運転 ('%s'),"Autopilot (""%s"")",Автопілот ('%s'),Piloto automático ('%s'),Автопилот („%s“),Pilote automatique ('%s')
-TRAIN_INFO_ENGINE,"Engines ('%s'; Check: Pantograph)","Motoren ('%s'; Pantograf überprüfen)",,,"Silniki (""%s""; Sprawdź: Pantograf)",,"Motores ('%s'; Chequea: Pantógrafo)","Двигатели („%s“)","Moteurs ('%s' ; Contrôle : Pantographe)"
+TRAIN_INFO_ENGINE,Engines ('%s'; Check: Pantograph),Motoren ('%s'; Pantograf überprüfen),,,"Silniki (""%s""; Sprawdź: Pantograf)",,Motores ('%s'; Chequea: Pantógrafo),Двигатели („%s“),Moteurs ('%s' ; Contrôle : Pantographe)
```
Comment (nalquas) [inline src/Translations/Ingame.csv]:
```suggestion
TRAIN_INFO_ENGINE,"Engines ('%s'; Check: Pantograph)","Motoren ('%s'; Pantograf überprüfen)",,,"Silniki (""%s""; Sprawdź: Pantograf)",,"Motores ('%s'; Chequea: Pantógrafo)","Двигатели („%s“)","Moteurs ('%s' ; Contrôle : Pantographe)"
```

**Review by nalquas — CHANGES_REQUESTED** [review]
Looks good to me, thanks.
However, please apply the suggested changes as the formatting of `Ingame.csv` has broken due to removing quotation marks.
Edit: I've taken the liberty to commit the changes myself.

**Review by nalquas — APPROVED** [review]
lgtm

**Comment by HaSa1002** [comment]
Thanks ❤️

**Comment by RacerBG** [comment]
> Looks good to me, thanks. However, please apply the suggested changes as the formatting of `Ingame.csv` has broken due to removing quotation marks. Edit: I've taken the liberty to commit the changes myself.

These were automatically made by LibreOffice, sorry about that.

**Pull request merged**